eventhubs/spring-cloud-azure-starter-integration-eventhubs/eventhubs-integration/README.md

```diff
@@ -0,0 +1,168 @@
+
+##### Add Role Assignment for Event Hubs
+
+1.  See [Managed identities for Azure resources with Event Hubs][role-assignment]
+    to add role assignment for Event Hubs. Assign `Contributor` role for managed identity.
+
+
+##### Update MSI related properties
+
+1.  Update [application-mi.yaml][application-mi.yaml].
+    ```yaml
+    spring:
+      cloud:
+        azure:
+          credential:
+            managed-identity-client-id: ${AZURE_MANAGED_IDENTITY_CLIENT_ID}
+          profile:
+            tenant-id: ${AZURE_TENANT_ID}
+          eventhubs:
+            namespace: ${AZURE_EVENTHUBS_NAMESPACE}
+            processor:
+              checkpoint-store:
+                container-name: ${AZURE_STORAGE_CONTAINER_NAME}
+                account-name:  ${AZURE_STORAGE_ACCOUNT_NAME}
+    ```
+    > We should specify `spring.profiles.active=mi` to run the Spring Boot application.
+    For App Service, please add a configuration entry for this.
+
+##### Redeploy Application
+
+If you update the `spring.cloud.azure.credential.managed-identity-client-id`
+property after deploying the app, or update the role assignment for
+services, please try to redeploy the app again.
+
+> You can follow
+> [Deploy a Spring Boot JAR file to Azure App Service][deploy-spring-boot-application-to-app-service]
+> to deploy this application to App Service
+
+## Examples
+
+1. Run the `mvn spring-boot:run` in the root of the code sample to get the app running.
+
+2. Send a POST request
+
+        $ ~~curl -X POST http://localhost:8080/messages?message=hello~~
+
+3. Verify in your app’s logs that a similar message was posted:
+
+        New message received: 'hello'
+        Message 'hello' successfully checkpointed
+
+4. Delete the resources on [Azure Portal][azure-portal] to avoid unexpected charges.
+
+
+## Troubleshooting
+
+## Next steps
+
+## Contributing
+
+
+<!-- LINKS -->
+[azure-account]: https://azure.microsoft.com/account/
+[azure-portal]: https://ms.portal.azure.com/
+[create-event-hubs]: https://docs.microsoft.com/azure/event-hubs/
+[create-azure-storage]: https://docs.microsoft.com/azure/storage/
+[create-managed-identity]: https://github.com/Azure-Samples/azure-spring-boot-samples/blob/spring-cloud-azure_4.0/create-managed-identity.md
+[create-sp-using-azure-cli]: https://github.com/Azure-Samples/azure-spring-boot-samples/blob/spring-cloud-azure_4.0/create-sp-using-azure-cli.md
+[eventhub-operation]: https://github.com/Azure/azure-sdk-for-java/blob/azure-spring-boot_3.6.0/sdk/spring/azure-spring-integration-eventhubs/src/spring-cloud-azure_4.0/java/com/azure/spring/integration/eventhub/api/EventHubOperation.java
+[receive-controller]: src/main/java/com/azure/spring/sample/eventhubs/ReceiveController.java
+[send-controller]: src/main/java/com/azure/spring/sample/eventhubs/SendController.java
+[application.yaml]: src/main/resources/application.yaml
+[application-sp.yaml]: src/main/resources/application-sp.yaml
```

Comment:
Don't use the file in main branch, replace it with the ones in spring-cloud-azure_4.0 branch. Same for other files

Reply:
Like to see the result for this relative path style.